packages/cashier/src/stores/__tests__/payment-agent-store.spec.js

```diff
@@ -13,8 +13,8 @@ describe('PaymentAgentStore', () => {
                 max_withdrawal: '2000',
                 min_withdrawal: '10',
                 email: 'pa@example.com',
-                telephone: '+12345678',
-                url: 'http://www.pa.com',
+                phone_numbers: '+12345678',
```

Comment:
Just for my understanding shouldn't these be `phone_number` and `url` because they are single string?

Reply:
@niloo-fs i have fixed the tests in the commit below. there were some changes from the BE -> phone_numbers and urls fields changed into arrays (`phone_numbers: [{phone_number: '123'}], urls: [{url: 'www'}]`). so now it looks right. big thanks for your notice/attention !

Reply:
Anytime, thank u for your explanations.